scapy/layers/vxlan.py

```diff
@@ -85,7 +85,6 @@ def mysummary(self):
 # By default, set both ports to the RFC standard
 bind_layers(UDP, VXLAN, sport=4789, dport=4789)
 
-bind_layers(VXLAN, Ether)
 bind_layers(VXLAN, IP, NextProtocol=1)
 bind_layers(VXLAN, IPv6, NextProtocol=2)
 bind_layers(VXLAN, Ether, flags=4, NextProtocol=0)
```

Comment:
@p-l- So this comes in useless ?

Reply:
Nope, this sets `flags=4` when building `VXLAN() / Ether()`.